### ISSUE TITLE
Activity Card - Scrolling Bug with Firefox on Mac

### ISSUE THREAD

**Issue by airforceixi** [JonnyWong16/plexpy#2221]
### Describe the Bug

The activity cards on the homepage won't scroll when using Firefox on a Mac. Tested with a blank Firefox profile, issue still exists.

Safari on Mac works fine. Firefox, Edge, and Chrome on Windows works fine. Firefox on Linux works fine.

### Steps to Reproduce

1. Start a stream with a hardware transcode.
2. View activity on home page using Firefox on Mac.

### Expected Behavior

Scrolling in activity card.

### Screenshots

![image](https://github.com/Tautulli/Tautulli/assets/32317421/30075f67-c274-4ca2-8bb0-f78e406da968)

### Tautulli Version

v2.13.4

### Git Branch

master

### Git Commit Hash

d019efcf911b4806618761c2da48bab7d04031ec

### Platform and Version

29) - Debian GNU/Linux 12 bookworm)[Docker] Linux 6.1.0-13-amd64 (#1 SMP PREEMPT_DYNAMIC Debian 6.1.55-1 (2023-09-

### Python Version

3.9.18 (main, Nov 29 2023, 05:59:34) [GCC 12.2.0]

### Browser and Version

Firefox on Mac v12.0.1

### Link to Logs

No logs

**Comment by beercity** [comment]
Just wanted to add an observation that this issue still exists in the latest release. I have no issue on Chrome under MacOS, this appears to be specific to Firefox.

**Comment by rcork** [comment]
This is still an issue. @JonnyWong16 why was this marked as completed?

**Comment by JonnyWong16** [comment]
Because it went stale since it is unreproducible.

**Comment by rcork** [comment]
I can confirm this issue still exists. Firefox on macos does not allow the activity card to scroll when video or audio wraps to a second line. This only happens on macos as i've tested Firefox on Windows 11 and Ubuntu and it doesn't not exhibit this issue. What other data can i submit so we can get this issue resolved?

**Comment by JonnyWong16** [comment]
Can anyone confirm if the issues is fixed or not on the `nightly` branch?

**Comment by rcork** [comment]
> Can anyone confirm if the issues is fixed or not on the `nightly` branch?

I’ll test out nightly tonight and report back

**Comment by rcork** [comment]
Confirmed that nightly branch fixes the issue

**Comment by rcork** [comment]
This bug has come back. Not sure if its because of a change Tautulli or Firefox made but with Firefox 153.0.4 on macOS 26.6.1 the activity card does not scroll when the bandwidth information is outside the card.

**Comment by zab1996** [comment]
Yeah this bug is back and happens on linux can't scroll on activity card or the watch stats. Also on Firefox

**Comment by rcork** [comment]
@JonnyWong16 do you want to reopen this bug or should we create a new issue to track it.

**Comment by JonnyWong16** [comment]
It's already issue #2776 and fixed in PR #2787.
